stubs/regex/@tests/stubtest_allowlist.txt

```diff
@@ -0,0 +1,4 @@
+regex._regex.Match
+regex._regex.Pattern
+regex._regex.Scanner
+regex._regex.Splitter
```

Comment:
This is only because there isn't actually a `regex/_regex.py` module so these classes aren't found at runtime. I'm using this file to store the classes defined in `regex/_regex.c`

Reply:
It would be good to add this as a comment to the allowlist :)

Reply:
Will do!